src/account-info/AccountDetails.js

```diff
@@ -31,28 +42,28 @@ export default class AccountDetails extends Component {
 
   render() {
     const { avatarUrl, fullName, email, status, auth } = this.props;
-
     return (
-      <View style={styles.padding}>
-        <View style={styles.avatarWrapper}>
-          <Avatar
-            avatarUrl={avatarUrl}
-            name={fullName}
-            size={100}
-            status={status}
-            realm={auth.realm}
+      <View style={styles.container}>
+        <Avatar
+          avatarUrl={avatarUrl}
+          name={fullName}
+          size={SCREEN_WIDTH}
```

Comment:
Facing one issue.
In landscape mode 
Current look
<img width="667" alt="screen shot 2017-06-12 at 11 09 57 am" src="https://user-images.githubusercontent.com/18511177/27020561-085a29e0-4f60-11e7-8c81-4a155ee7ba8f.png">
avatar is stretched to full screen and page is not scrollable.
How about creating a different UI for portrait and landscape.
- in portrait (current is awesome 👍 )
- in landscape we can have avatar in the left and other details (name, button etc) in the right. (making right section scrollable if required) (somewhat like WhatsApp)